### PR TITLE
feature/first-scaffolding

### PR DESCRIPTION
**Pull request by KennethanCeyer**
# Features

- Add gatsby basic scaffolding (emotion + typescript + gatsby)
- Add basic README.md documentation

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+      // Used to generate URL to view this content.
+      createNodeField({
+        node,
+        name: 'slug',
+        value: slug || ''
+      })
+
+      // Used to determine a page layout.
+      createNodeField({
```
Comment (KennethanCeyer) [inline gatsby-node.js:32]: Review test

Comment (Hansanghyeon) [reply gatsby-node.js:32]: comment test

Comment (devGW) [reply gatsby-node.js:32]: comment test

**Comment by KennethanCeyer** [comment]
@Hansanghyeon 
Which is mean `deploy`?
Is this mean about deployment of the source code or blog contents?

I think both of the ways to manage should be in the repository

**Comment by KennethanCeyer** [comment]
Could any folks please review this?

**Pull request merged**